LAPACKE/include/lapack.h

```diff
@@ -80,7 +81,11 @@ extern "C" {
 
 /*----------------------------------------------------------------------------*/
 #ifndef lapack_int
-#define lapack_int     int
+#define lapack_int     int32_t
```

Comment:
I think we should use:

```C
#ifndef lapack_int
#if defined(LAPACK_ILP64)
#define lapack_int              int64_t
#else
#define lapack_int              int32_t
#endif
#endif
```

for `lapack_int` as in [LAPACKE/include/lapacke_config.h](https://github.com/Reference-LAPACK/lapack/pull/700/files#diff-7db5a114acf1e298d576cd7998665526f604cfd7d0b50ce8121dd321507abee1)

Reply:
Yes! That way users don't have to define HAVE_LAPACK_CONFIG_H anymore if they want a 64-bit integer build with the standard config.